### DISCUSSION TITLE
Renaming default results structure

### DISCUSSION THREAD

**Discussion by BumpaRoy** [Azure/data-api-builder#1764] [Q&A] [answered]
In the GraphQL query below, is there a way to not use the "items" identifier and use my own.  So the first example I'd want to be "parts" and the second one "locations".
![image](https://github.com/Azure/data-api-builder/assets/137801704/9d635b88-f6e9-48a0-8c2a-87a0d40cea3a)

**Comment by yorek** [comment, marked as answer]
Hi BumpaRoy

It is not possible to change to name `items` to something else in the GraphQL query, but if you need to have the `items` element using another name in the returned result, you can use aliasing to do that. For example:

```graphql
query {
  customers {
    contacts: items  {
      CustomerID
      Title
      FirstName
      LastName
      EmailAddress
    }
  }
}
```

would return the following result:

```json
{
  "data": {
    "customers": {
      "contacts": [
        {
          "CustomerID": 1,
          "Title": "Mr.",
          "FirstName": "Orlando",
          "LastName": "Gee",
          "EmailAddress": "orlando0@adventure-works.com"
        }
      ]
    }
  }
}
```
